tools/es5to6.json

```diff
@@ -1,121 +1,112 @@
 {
-  "Francis Carr": [],
-  "Jon Norman": [],
   "Regis Kuckaertz": [
-    "projects/common/modules/discussion/comment-count.js",
-    "projects/common/modules/discussion/comments.js",
-    "projects/common/modules/discussion/discussion-frontend.js"
-  ],
-  "Richard Nguyen": [],
-  "Kate Whalen": [],
-  "sndrs": [
+    "projects/common/modules/atoms/quiz.js",
     "bootstraps/enhanced/accessibility.js",
     "bootstraps/enhanced/common.js",
     "bootstraps/enhanced/crosswords.js",
-    "bootstraps/enhanced/facia.js",
-    "bootstraps/enhanced/football.js",
-    "projects/common/modules/preferences/main.js",
-    "bootstraps/enhanced/gallery.js"
-  ],
-  "Simon Adcock": [
-    "projects/common/modules/experiments/tests/paid-content-vs-outbrain.js",
-    "projects/common/modules/ui/clickstream.js",
-    "projects/common/modules/discussion/user-avatars.js",
-    "projects/common/modules/discussion/whole-discussion.js"
-  ],
-  "Nicolas Long": [
-    "projects/common/modules/ui/selection-sharing.js",
-    "bootstraps/enhanced/profile.js",
-    "bootstraps/enhanced/recipe-article.js",
-    "bootstraps/enhanced/sport.js",
-    "bootstraps/enhanced/trail.js",
-    "bootstraps/enhanced/youtube.js",
-    "bootstraps/video-embed.js",
-    "bootstraps/youtube-embed.js",
-    "lib/atob.js",
-    "projects/common/modules/component.js"
-  ],
-  "Calum Campbell": [
-    "lib/picturefill.js",
-    "lib/proximity-loader.js",
-    "projects/common/modules/video/metadata.js"
-  ],
-  "GHaberis": [
-    "projects/common/modules/discussion/activity-stream.js",
-    "projects/common/modules/discussion/api.js",
-    "projects/common/modules/discussion/comment-box.js"
-  ],
-  "Gustav Pursche": [
-    "projects/common/modules/discussion/comment-box.js",
-    "projects/common/modules/video/onward-container.js"
-  ],
-  "NataliaLKB": [
     "projects/common/modules/article/rich-links.js",
     "projects/common/modules/article/twitter.js",
     "projects/common/modules/asyncCallMerger.js",
-    "projects/common/modules/atoms/quiz.js",
-    "projects/common/modules/atoms/story-questions.js",
-    "projects/common/modules/atoms/youtube-tracking.js"
+    "projects/common/modules/ui/clickstream.js",
+    "projects/common/modules/ui/expandable.js",
+    "projects/common/modules/ui/faux-block-link.js"
   ],
-  "Gideon Goldberg": [],
-  "Matthew Walls": [
+  "sndrs": [
+    "bootstraps/enhanced/facia.js",
+    "bootstraps/enhanced/football.js",
+    "bootstraps/enhanced/gallery.js",
     "projects/common/modules/crosswords/cell.js",
     "projects/common/modules/crosswords/classNames.js",
     "projects/common/modules/crosswords/clues.js",
     "projects/common/modules/crosswords/comments.js",
     "projects/common/modules/crosswords/confirm-button.js",
+    "projects/common/modules/discussion/activity-stream.js",
+    "projects/common/modules/discussion/api.js"
```

Comment:
Cool, I'll bump all your discussion stuff to the top since you're on a roll! (but feel free to amend the order as you see fit)